circuits/cpp/src/aztec3/circuits/rollup/test_utils/utils.cpp

```diff
@@ -351,6 +350,9 @@ RootRollupInputs get_root_rollup_inputs(utils::DummyBuilder& builder,
     MemoryStore l1_to_l2_msg_tree_store;
     MerkleTree l1_to_l2_msg_tree(l1_to_l2_msg_tree_store, L1_TO_L2_MSG_TREE_HEIGHT);
 
+    MemoryStore historic_blocks_tree_store;
+    MerkleTree historic_blocks_tree(historic_blocks_tree_store, HISTORIC_BLOCKS_TREE_HEIGHT);
+
```

Comment:
Should probably be initialised with empty root for position 0 like we were doing with the others.

Reply:
fixing in follow up